build/storage/scripts/disk_infrastructure.py

```diff
@@ -0,0 +1,207 @@
+import base64
+import importlib
+import logging
+import re
+import socket
+import sys
+import time
+import uuid
+
+from scripts import socket_functions
+
+sys.path.append("/usr/libexec/spdk/scripts")
```

Comment:
is the installation path explicitly defined in the container?

Reply:
It is defined explicitly in a file post-install,
path: build/storage/build_base/post-install
`# Be nice for docker exec and link SPDK scripts|binaries under common PATH
# location like /usr/bin.
for f in /usr/local/bin/*; do ln -sf "$f" /usr/bin ; done
for f in /usr/local/bin/fio/*; do ln -sf "$f" /usr/bin ; done
ln -s /usr/libexec/spdk/scripts/rpc.py /usr/bin
ln -s /usr/libexec/spdk/scripts/rpc_http_proxy.py /usr/bin
ln -s /usr/libexec/spdk/scripts/setup.sh /usr/bin
ln -s /usr/libexec/spdk/include/spdk /usr/include
ln -s /usr/libexec/spdk/scripts/ /usr
ln -s /usr/libexec/spdk/scripts/sma.py /usr/bin
ln -s /usr/libexec/spdk/scripts/sma-client.py /usr/bin`